VENCore.podspec

```diff
@@ -1,19 +1,14 @@
 Pod::Spec.new do |s|
-  s.name         = "VENCore"
-  s.version      = "3.1.1"
-  s.summary      = "Core Venmo client library"
-  s.description  = <<-DESC
-                   Core iOS client library for the Venmo api
-                   DESC
-
-  s.homepage     = "https://github.com/venmo/VENCore"
+  s.name         = 'VENCore'
+  s.version      = '3.1.2'
+  s.summary      = 'Core Venmo client library'
+  s.description  = 'Core iOS client library for the Venmo api'
+  s.homepage     = 'https://github.com/venmo/VENCore'
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.author       = { "Venmo" => "ios@venmo.com" }
+  s.author       = { 'Venmo' => 'ios@venmo.com' }
   s.platform     = :ios, '6.0'
-  s.source       = { :git => "https://github.com/venmo/VENCore.git",
-                     :tag => "v#{s.version}"
-  }
+  s.source       = { :git => 'https://github.com/venmo/VENCore.git',
+                     :tag => "v#{s.version}" }
   s.source_files = 'VENCore/**/*.{h,m}'
-  s.dependency 'CMDQueryStringSerialization', '~>0.2.0'
-  s.requires_arc = true
+  s.dependency 'CMDQueryStringSerialization'
```

Comment:
I'm kind of OK either way. It's a hassle to update all the Pods with every version bump. 
Also, technically since the major version is 0, there can be breaking changes in any version updates.

@calebd if the public API is finalized and permanent, any reason not to release `1.0.0`? :octocat: